prow/jobs/incubator/reconciler/reconciler.yaml

```diff
@@ -275,6 +275,79 @@ periodics: # runs on schedule
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: reconciler-control-plane-image-bump
+      annotations:
+        description: "Reconciler image bump in control-plane"
+        pipeline.trigger: "periodic"
+        testgrid-create-test-group: "false"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "reconciler-control-plane-image-bump"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-log-collector-slack-token: "true"
+      cron: "0 1-22/4 * * 1-5"
+      skip_report: false
+      decorate: true
+      decoration_config:
+        grace_period: 600000000000
+        timeout: 14400000000000
+      cluster: untrusted-workload
+      extra_refs:
+        - org: kyma-project
+          repo: control-plane
+          path_alias: github.com/kyma-project/control-plane
+          base_ref: main
+        - org: kubernetes
+          repo: test-infra
+          path_alias: github.com/kubernetes/test-infra
+          base_ref: master
+          base_sha: 3fba7110c39a7a0e7fb087dbb4af0674cafd341c
+        - org: kyma-incubator
+          repo: reconciler
+          path_alias: github.com/kyma-incubator/reconciler
+          base_ref: main
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
```

Comment:
Does this job still need a reference to the test-infra repo if it's been moved?

Reply:
I have moved the autobump config file to reconciler and removed this reference for test-infra.